platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegionStatus.java

```diff
@@ -4,53 +4,19 @@
  * A region's status includes its active/inactive state as well as counts
  * of the number of resources that have completed downloading, their total
  * size in bytes, and the total number of resources that are required.
- *
+ * <p>
  * Note that the total required size in bytes is not currently available. A
  * future API release may provide an estimate of this number.
  */
```

Comment:
also need to close the tag, I recall this showing as a warning or error when generating javadoc.